Jenkinsfile

```diff
@@ -191,18 +176,18 @@ pipeline {
                 }
             }
         }
-        stage('Deploy JupyterHub to NCATS') {
-            agent {
-                node { label 'ls-api-ci.ncats' }
-            }
-            steps {
-                configFileProvider([configFile(fileId: 'env-single-node', targetLocation: '.env')]) {
-                    withKubeConfig([credentialsId: 'ncats_polus2']) {
-                        sh "bash ./deploy.sh"
-                    }
-                }
-            }
-        }
+        // stage('Deploy JupyterHub to NCATS') {
```

Comment:
Should you remove?

Reply:
Yeah, I disabled this deployment, because it is not functioning, but @sunnielyu wanted to keep it around in case it comes back again. I am leaning to remove it for now and use the older commits for reference of what it was. We can always add it back if and when it's needed